test/sagemaker_tests/tensorflow/tensorflow2_training/integration/conftest.py

```diff
@@ -155,6 +156,9 @@ def processor():
         return os.environ['TEST_PROCESSORS'].split(',')
     return None
 
+@pytest.fixture(scope='session', name='multi_region_support')
```

Comment:
Please return `None` for the fixture `n_virginia_sagemaker_session` and `n_virginia_ecr_image` if `not multi_region_support`, as an input guard. This will ensure that regions in other partitions do not fail, looking for access to `us-east-1`.

Reply:
Ack